.ci/build-mac-arm64.sh

```diff
@@ -11,19 +11,11 @@ BREW_PATH="$(brew --prefix)"
 export BREW_BIN="/opt/homebrew/bin"
 export BREW_SBIN="/opt/homebrew/sbin"
 export CMAKE_EXTRA_OPTS='-DLLVM_TARGETS_TO_BUILD=AArch64;ARM'
-
-
-export VULKAN_SDK
-VULKAN_SDK"=$(brew --prefix)/opt/molten-vk"
-ln -s "$VULKAN_SDK/lib/libMoltenVK.dylib" "$VULKAN_SDK/lib/libvulkan.dylib"
-export VK_ICD_FILENAMES
-VK_ICD_FILENAMES="$VULKAN_SDK/share/vulkan/icd.d/MoltenVK_icd.json"
 export Qt5_DIR
 Qt5_DIR="$(brew --prefix)/opt/qt5"
-export SDL2_DIR
-SDL2_DIR="$BREW_PATH/opt/sdl2/lib/cmake/SDL2"
-export LLVM_DIR
-LLVM_DIR="$(brew --prefix)/opt/llvm@16"
+export PATH="$BREW_PATH/opt/llvm@16/bin:$BREW_BIN:$BREW_SBIN:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Apple/usr/bin:$PATH"
+export LDFLAGS="-L$BREW_PATH/lib -Wl,-rpath,$BREW_PATH/lib"
+export CPPFLAGS="-I$BREW_PATH/include -msse -msse2 -mcx16 -no-pie"
```

Comment:
I don’t really understand things to this depth, but I would have thought that there are no SSE or SSE2 or any other x86 extensions on arm…?

Reply:
I don't fully understand it either im just experimenting